server/app/views/admin/migration/AdminExportViewPartial.java

```diff
@@ -0,0 +1,66 @@
+package views.admin.migration;
+
+import static j2html.TagCreator.button;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.form;
+
+import com.google.inject.Inject;
+import controllers.admin.routes;
+import j2html.tags.DomContent;
+import j2html.tags.specialized.DivTag;
+import play.mvc.Http;
+import views.BaseHtmlView;
+import views.components.ButtonStyles;
+import views.components.FieldWithLabel;
+
+/** An HTMX partial for portions of the page rendered by {@link AdminExportView}. */
+public final class AdminExportViewPartial extends BaseHtmlView {
+  @Inject
+  AdminExportViewPartial() {}
```

Comment:
suggestion (non-blocking):  I don't think you need this constructor since it's the default anyway.

Reply:
removed